schema/classes/Asset.yaml

```diff
@@ -0,0 +1,52 @@
+"@context":
+  schema: http://schema.org/
+  rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+  rdfs: http://www.w3.org/2000/01/rdf-schema#
+  xsd: http://www.w3.org/2001/XMLSchema#
+  dandi: http://dandiarchive.org/schema/
+  crypto: http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions/
+  md5: cyrpto:md5
+  sha1: crypto:sha1
+  sha256: crypto:sha256
+  sha512: crypto:sha512
```

Comment:
Looks like the BLAKE family of functions do not have pages at http://id.loc.gov/vocabulary/preservation/cryptographicHashFunctions/.

Reply:
we can add those from elsewhere, but we are actually going to only enforce/validate only a couple of these for dandi.

Reply:
Sounds good. Just wanted to make sure that there was a source for identifying other algorithms since I imagine that at some point fast cryptographic hashing of very big files is going to be a requirement even if it won't be in the spec right now.